### ISSUE TITLE
doesn't work at all

### ISSUE THREAD

**Issue by chriz74x** [Flynsarmy/laravel-csv-seeder#8]
I am trying this one to seed my DB from CSV.
I run it and nothing happens.
Also please could you explain exactly the syntax used to set the delimiter?
Can you also explain what happens to the first line, the header, in the csv?
Is it used to set the columns for seeding in the table?
Is it used at all?
I mean, what does the package expects from the csv file? Same number or columns? as in the db table?

**Comment by Flynsarmy** [comment]
> explain exactly the syntax used to set the delimiter?

As per the readme, they can be set in your seeder class constructor:
```php
public function __construct()
{
    $this->table = 'gtfs_stops';
    $this->csv_delimiter = ',';
    $this->filename = app_path().'/database/seeds/csvs/stops.txt';
}
```

> Can you also explain what happens to the first line, the header, in the csv? Is it used to set the columns for seeding in the table? 

Yes, it's used to set the table columns your CSV columns are mapped to. So a CSV of
```
id,name
1,Foo
2,Bar
```

will be mapped to id and name columns in your DB. You can specify some or all DB columns. It will only import what you specify.

> I am trying this one to seed my DB from CSV. I run it and nothing happens.

Change app_path() to base_path() in your constructor and drop your csv into */database/seeds/csvs/yourcsvfile.csv*. Works fine after that.

**Comment by Flynsarmy** [comment]
I've released version 1.0.4 which will log missing/unreadable CSV files during import.

**Comment by psutinrerk** [comment]
I have the same problem 1.0.4, nothing happen, no error log and I used "base_path()"

Could you please help :(

I look in to /storage/logs/laravel.log and can't find any error related to this

thank you
